### DISCUSSION TITLE
Maximum sending rate exceeded when sending emails

### DISCUSSION THREAD

**Discussion by umerghori786** [aws/aws-php-sns-message-validator#59] [Q&A]
Hi i am using amazon ses service when i send 200 emails from my local server it sent all email perfectly and jobs table is empty but when i sent 200 emails on live server it send only 90 email and remaining data are added in failed_jobs table.. it show this error in exception column
exception 'Aws\Ses\Exception\SesException' with message 'Error executing "SendEmail" on "https://email.us-east-1.amazonaws.com"; AWS HTTP error: Client error: `POST https://email.us-east-1.amazonaws.com` resulted in a `400 Bad Request` response:
<ErrorResponse xmlns="http://ses.amazonaws.com/doc/2010-12-01/">
  <Error>
    <Type>Sender</Type>
    <Code>Throttling< (truncated...)
 Throttling (client): Maximum sending rate exceeded. - <ErrorResponse xmlns="http://ses.amazonaws.com/doc/2010-12-01/">
  <Error>
    <Type>Sender</Type>
    <Code>Throttling</Code>
    <Message>Maximum sending rate exceeded.</Message>
  </Error>
  <RequestId>8f23e6e6-b4f2-4572-b426-343b370cb32c</RequestId>
</ErrorResponse>
'

**Comment by mfn** [comment]
https://aws.amazon.com/premiumsupport/knowledge-center/ses-sending-rate-quota-exceeded/
